extensions/mssql/src/utils.ts

```diff
@@ -151,6 +169,9 @@ export function getCommonLaunchArgsAndCleanupOldLogFiles(logPath: string, fileNa
 	console.log(`Old log files deletion report: ${JSON.stringify(deletedLogFiles)}`);
 	launchArgs.push('--tracing-level');
 	launchArgs.push(getConfigTracingLevel());
+	if (getConfigPiiLogging()) {
+		launchArgs.push('--piiLogging');
```

Comment:
This should be a runtime toggle, we shouldn't be continuing to collect PII if the user has this turned off but hasn't reloaded the app yet